src/main/java/com/animalfinder/AnimalFinderController.java

```diff
@@ -1,14 +1,14 @@
 package com.animalfinder;
 
-import java.awt.List;

```

Comment:
This is definitely the wrong import... java.awt.List is a UI component that was deprecated back in 1999... I think you mean to use java.util.List.